ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-file-answer/activityFileAnswer.component.ts

```diff
@@ -127,6 +127,18 @@ export class ActivityFileAnswer implements OnInit, OnDestroy {
         this.ngUnsubscribe.complete();
     }
 
+    validateFiles(files: File[]): boolean {
```

Comment:
Making a method public just for passing tests is not a nice idea.
See[ how avoid it](https://github.com/broadinstitute/ddp-angular/pull/1149/files#r838720436)